plugins/scaffolder-backend/api-report.md

```diff
@@ -721,6 +736,10 @@ export type TaskCompletionState = 'failed' | 'completed';
 
 // @public
 export interface TaskContext {
+  // (undocumented)
+  abortSignal: AbortSignal;
```

Comment:
If the wording everywhere else is "cancel", is it worth calling this `cancelSignal`? I'm not sure which is more confusing :)